decidim-accountability/spec/controllers/decidim/accountability/admin/import_results_controller_spec.rb

```diff
@@ -10,7 +10,13 @@ module Admin
 
         let(:organization) { create(:organization) }
         let(:current_user) { create(:user, :confirmed, :admin, organization: organization) }
-        let(:params) { { participatory_process_slug: participatory_space.slug, component_id: component.id } }
+        let(:params) do
+          {
+            script_name: "/admin/participatory_process/#{participatory_space.slug}",
```

Comment:
@ahukkanen , this is not suitable. As soon as I am implementing the recommendation, the tests are starting to fail due to fact the Frontend route is matched instead of admin one.

Reply:
@ahukkanen : 
- https://github.com/decidim/decidim/runs/5495318271?check_suite_focus=true#step:5:133
- https://github.com/decidim/decidim/runs/5495318457?check_suite_focus=true#step:5:326
- https://github.com/decidim/decidim/runs/5495318387?check_suite_focus=true#step:5:327
-

Reply:
Thanks @alecslupu !

I think we need to duplicate the original default URL options, this should work:

```ruby
module Decidim
  module ControllerExampleGroup
    extend ActiveSupport::Concern

    class_methods do
      def routes
        before do
          routes = yield
          @orig_default_url_options = routes.default_url_options.dup # <= note .dup
          routes.default_url_options[:script_name] = ""

          self.routes = routes
        end

        after do
          routes.default_url_options = @orig_default_url_options
        end
      end
    end
  end
end

RSpec.configure do |config|
  config.include Decidim::ControllerExampleGroup, type: :controller
end
```

Reply:
Considering the jobs output, i would leave it as it is.

Reply:
The jobs output was because the `default_url_options` was changed permanently for the engine. When you do the above, it is reverted back to the original after the controller test.

I tested the above and run the controller + failed helper tests consecutively in the same run in those linked jobs with this and it worked fine.

This is only needed for the controller tests and I would really suggest doing this, because this also affects controller tests outside of the core. The less changes we need when upgrading, the better IMO.